packages/renderer/src/components/Button/component.tsx

```diff
@@ -1,18 +1,63 @@
+import { useCallback, useState } from 'react';
 import cx from 'classnames';
-import { Button } from 'decentraland-ui2';
+import { Button as _Button, ButtonGroup as _ButtonGroup } from 'decentraland-ui2';
```

Comment:
wdyt about using the prefix DCL rather than `_`?
```suggestion
import { Button as DCLButton, ButtonGroup as DCLButtonGroup } from 'decentraland-ui2';
```

Reply:
yes!